### ISSUE TITLE
Reference link to not a link

### ISSUE THREAD

**Issue by zakiego** [senar-ai/web#17]
## Description

The problem starts when the link field is not filled with a link

![image](https://user-images.githubusercontent.com/78015359/173460212-35dafbd5-d22b-43e9-8df8-9e8139c16e0c.png)

![image](https://user-images.githubusercontent.com/78015359/173460718-6342bb0f-5b6f-49f5-80ba-237a6357cdb1.png)

![Screenshot from 2022-06-14 06-46-15](https://user-images.githubusercontent.com/78015359/173460323-3f00b403-3c9e-49f4-afd6-dbb00f83cd9d.png)

## Expected Behaviour

Some possible solutions:
1. Filter link column on backend/frontend
2. Make end to end testing to ensure all links can be clicked

**Comment by zainfathoni** [comment]
Thanks for the bug report!
As a temporary fix in production, I just fixed the data to provide a proper link to the row.
https://senar.ai/activities/semua-usia

I agree that we should write more tests to make the website more resilient.
Once I'm done adding more test scripts to https://github.com/zainfathoni/pkode, I will start writing both unit and e2e tests.

In the meantime, let's keep this issue open as a reminder to cover this use case in the tests.

**Comment by zakiego** [comment]
I just saw this pull request https://github.com/senar-ai/web/pull/20

Very simple setup, amazing 😃👍🏻

**Comment by zainfathoni** [comment]
Yeah, I have been working on https://github.com/zainfathoni/pkode for the past week. I'm aiming to finish the basic functionality this weekend and get back to improving this project soon.
